pacman_vision: 
  {
    Master_Disable: false,
    cluster_tol: 0.05,
    color_importance: 0.3333,
    crop_left_arm: true,
    crop_left_hand: false,
    crop_right_arm: true,
    crop_right_hand: false,
    downsample_leaf_size: 0.005,
    downsampling: false,
    enable_broadcaster: true,
    enable_estimator: false,
    enable_listener: true,
    enable_supervoxels: false,
    enable_tracker: false,
    estimated_objects: true,
    estimation_type: 0,
    iterations: 5,
    keep_organized: false,
    external_kinect2_resolution: 1,
    neighbors: 20,
    normal_importance: 0.3333,
    normals_search_radius: 0.01,
    object_calibration: false,
    pass_xmax: 0.2,
    pass_xmin: -0.2,
    pass_ymax: 0.1,
    pass_ymin: -0.3,
    pass_zmax: 1.0,
    pass_zmin: 0.4,
    passthrough: true,
    passthrough_limits: true,
    plane_segmentation: false,
    plane_tolerance: 0.01,
    publish_tf: true,
    refinement_iterations: 2,
    seed_resolution: 0.18,
    spatial_importance: 0.3333,
    tracker_bounding_box: false,
    tracker_disturbance: false,
    sensor_type: 0,
    use_service: false,
    use_table_transform: false,
    voxel_resolution: 0.02
  }
